internal/venice-common/src/main/java/com/linkedin/venice/controller/converters/ResponseConverter.java

```diff
@@ -0,0 +1,14 @@
+package com.linkedin.venice.controller.converters;
+
+import com.linkedin.venice.controllerapi.ControllerResponse;
+
+
+/**
+ * A general interface to represent response converters that operates on transport specific response {@Link T} to produce
+ * a controller specific responesn {@link D}
+ * @param <T> Source transport specific response type
+ * @param <D> Controller response type
```

Comment:
If you decide to change the generic type names in the above interface, then let's adopt the same convention here.

Reply:
Started a thread above and will reconcile this one once we land on something.

Reply:
Will follow the same guidelines as the above comment.